### PR TITLE
More Archiver fixes

### PR DESCRIPTION
**Pull request by samchrisinger**

### CODE REVIEW

File: website/discovery/views.py [hunk 1]
```diff
@@ -2,12 +2,19 @@
...
 from website import settings
 from website.project import Node
+from website.project.utils import recent_public_registrations
 
 from modularodm.query.querydialect import DefaultQueryDialect as Q
 
 from framework.analytics.piwik import PiwikClient
 
 
+recent_query = (
```
Comment (sloria) [inline website/discovery/views.py]: This doesn't seem to be used and is reassigned on ~L50

Comment (sloria) [inline website/discovery/views.py]: You might want to this in `website.project.utils` and import it in this module.

**Comment by sloria** [comment]
Pass finished. :ear:

**Comment by lbanner** [comment]
@sloria .... I believe this is an answer to our '500 Server Error' obstacle .... can we review and merge? Thanks.

File: website/archiver/tasks.py [hunk 2]
```diff
@@ -212,15 +212,19 @@ def archive_node(results, job_pk):
...
             if not result.num_files:
                 job.update_target(result.target_name, ARCHIVER_SUCCESS)
                 continue
+            addons_archived = addons_archived + 1
             archive_addon.delay(
                 addon_short_name=result.target_name,
                 job_pk=job_pk,
                 stat_result=result,
             )
+        if not addons_archived:
```
Comment (sloria) [inline website/archiver/tasks.py]: Why the `not`? Shouldn't the signal be sent if there were addons archived?

Comment (samchrisinger) [inline website/archiver/tasks.py]: No, that signal gets sent in the archive_addon callbacks.

This ensures logs are added even if there are no addons being archived.

**Pull request merged**